package.json

```diff
@@ -43,17 +43,19 @@
     "test-node-coverage-html": "nyc --reporter=lcov --reporter=html mocha --recursive test/a/*  */pp test/modules ts/test libloki/test/node",
     "eslint": "eslint .",
```

Comment:
We can also enable cache for eslint (save 4-5 seconds)
Did not find the equivalent for tslint though

Reply:
Oh cool, will add